src/lsp/lp_lsp.ml

```diff
@@ -391,6 +450,10 @@ let dispatch_message ofmt dict =
     protect_dispatch "didClose"
       (do_close ofmt) params
 
+  | "proof/debugFlags" ->
+    protect_dispatch "debugFlags"
+      (do_debugFlags ofmt) params
+
```

Comment:
Could you please explain what is the goal of this message? Is it in the LSP standard?

Reply:
This is used to change the debug flags. It is not in the LSP standard.